integration/simple-cache-client.test.ts

```diff
@@ -386,3 +375,1633 @@ describe('Integration Tests for operations on sets datastructure', () => {
+    );
+    expect(response).toBeInstanceOf(CacheDictionaryIncrement.Success);
+    await new Promise(r => setTimeout(r, 4900));
+
+    response = await momento.dictionaryGetField(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field
+    );
+    const getResponse = await momento.dictionaryGetField(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field
+    );
+    expect(getResponse).toBeInstanceOf(CacheDictionaryGetField.Miss);
+  });
+
+  it('should dictionaryIncrement with refresh ttl', async () => {
+    const dictionaryName = v4();
+    const field = v4();
+    let response = await momento.dictionaryIncrement(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field,
+      undefined,
+      CollectionTtl.of(2)
+    );
+    expect(response).toBeInstanceOf(CacheDictionaryIncrement.Success);
+
+    response = await momento.dictionaryIncrement(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field,
+      undefined,
+      CollectionTtl.of(10)
+    );
+    expect(response).toBeInstanceOf(CacheDictionaryIncrement.Success);
+    await new Promise(r => setTimeout(r, 2000));
+
+    const getResponse = await momento.dictionaryGetField(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field
+    );
+    expect(getResponse).toBeInstanceOf(CacheDictionaryGetField.Hit);
+    if (getResponse instanceof CacheDictionaryGetField.Hit) {
+      expect(getResponse.valueString()).toEqual('2');
+    }
+  });
+
+  it('should dictionaryIncrement successfully with setting and resetting field', async () => {
+    const dictionaryName = v4();
+    const field = v4();
+
+    await momento.dictionarySetField(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field,
+      '10'
+    );
+    let response = await momento.dictionaryIncrement(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field,
+      0
+    );
+    expect(response).toBeInstanceOf(CacheDictionaryIncrement.Success);
+    let successResponse = response as CacheDictionaryIncrement.Success;
+    expect(successResponse.valueNumber()).toEqual(10);
+
+    response = await momento.dictionaryIncrement(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field,
+      90
+    );
+    expect(response).toBeInstanceOf(CacheDictionaryIncrement.Success);
+    successResponse = response as CacheDictionaryIncrement.Success;
+    expect(successResponse.valueNumber()).toEqual(100);
+
+    // Reset field
+    await momento.dictionarySetField(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field,
+      '0'
+    );
+    response = await momento.dictionaryIncrement(
+      INTEGRATION_TEST_CACHE_NAME,
+      dictionaryName,
+      field,
+      0
+    );
+    expect(response).toBeInstanceOf(CacheDictionaryIncrement.Success);
+    successResponse = response as CacheDictionaryIncrement.Success;
+    expect(successResponse.valueNumber()).toEqual(0);
+  });
```

Comment:
I have been puzzled by this part of the test keeps failing...
`dictionaryIncrement` succeeds but the passed value to `CacheDictionaryIncrement.Success` is somehow `undefined`.
This seems to only happen when I try to increment 0 by 0.
When I tried to increment 0 by 1 and increment 1 by 0, both succeeded.
Somehow only JS cannot increment 0 by 0 (there was no issue for the same test in the .NET and PHP SDKs).

Reply:
PHP SDK test: https://github.com/momentohq/client-sdk-php/blob/main/tests/Cache/CacheClientTest.php#L1204
.NET SDK test: https://github.com/momentohq/client-sdk-dotnet-incubating/blob/main/tests/Integration/Momento.Sdk.Incubating.Tests/DictionaryTest.cs#L210

Reply:
@poppoerika Was this fixed? If not, can we make a FF issue?